src/asm/string.c

```diff
@@ -0,0 +1,51 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   string.c                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: asarandi <asarandi@student.42.us.org>      +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2018/02/11 20:50:27 by asarandi          #+#    #+#             */
+/*   Updated: 2018/03/01 23:37:33 by asarandi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "asm.h"
+
+bool	string_is_integer(char *str)
+{
+	int		i;
+	bool	result;
+
+	i = 0;
+	result = false;
+	if (str[i] == '-')
+		i++;
+	while (str[i])
+	{
+		if (!ft_isdigit(str[i]))
+		{
+			result = false;
+			break ;
+		}
+		result = true;
+		i++;
+	}
+	return (result);
```

Comment:
@riking Could a simple ft_atoi check if errno returns an nonzero value?
https://github.com/riking/libft/blob/master/ft_atoi.c#L55

That might be a solid way to see if the string is in fact, an integer?